input/templates/partials/catalog/reserve/map.hbs

```diff
@@ -1,62 +1,46 @@
 <div class="map">
   <div id="map"></div>
 </div>
-
 <script>
     var map;
-
     function initialize() {
         // creates the map with default options
         map = new google.maps.Map(document.getElementById('map'), {
             zoom: 11,
-            center: new google.maps.LatLng(48.1545702, 11.5616558), // find out current position
+            center: new google.maps.LatLng(48.1545702, 11.5616558), // FIXME: here needs to be the "city/country/place" where it will be centered at first (this is a mock lat for Munich)
```

Comment:
@lufego is it possible to obtain the initial location by the browser's location info, and in case it's missing then center in the world zoomed out?